STX.REST.RESTFulSense.Clients/Services/Foundations/HttpExchanges/HttpExchangeService.cs

```diff
@@ -55,16 +55,18 @@ await httpBroker.SendRequestAsync(
                 httpResponseMessage);
         });
 
-        public async ValueTask<HttpExchange> PostAsync(
+        public ValueTask<HttpExchange> PostAsync(
             HttpExchange httpExchange,
-            CancellationToken cancellationToken = default)
+            CancellationToken cancellationToken = default) => TryCatch(async () =>
```

Comment:
TryCatch should be in the next line

Reply:
Remove tab from TryCatch. See how its done here

![image](https://github.com/user-attachments/assets/99771bbf-2fa0-4ac9-a689-4e3e33d7b139)

Reply:
I fixed